files/common/var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -445,3 +445,12 @@
 - command: pam-auth-update --enable challenge-response
   when:
     - variant is regex("external-.*")
+
+#
+# Configure kdump so that makedumpfile excludes ZFS ARC file data pages
+#
+- lineinfile:
+    path: /etc/default/kdump-tools
+    backrefs: yes
+    regexp: '^#MAKEDUMP_ARGS=.*$'
```

Comment:
Should we add `?` after the `#` in `regexp` so that the line gets replaced/updated even after we make the initial change? i.e.
```
    regexp: '^#?MAKEDUMP_ARGS=.*$'
```
otherwise, after we make this change the first time, if we change the `line` below in a later update, the new configuration won't take effect, right? due to the `#` character having been removed.

I think it'd be better to configure this `regexp` field so that the latest configuration will always take effect, even if an earlier configuration is in place (by adding the `?` character).

Also, I don't think we need to have the `.*$` part.. the whole line will be replaced, even if only the start of the line is matched by `regexp`.

For an example of this, see the `sshd_config` changes we make earlier in this file.